src/otelcollector/otelcol-config.yml

```diff
@@ -21,6 +21,58 @@ receivers:
     endpoint: "valkey-cart:6379"
     username: "valkey"
     collection_interval: 10s
+  # Host metrics
+  hostmetrics:
+    root_path: /hostfs
+    scrapers:
```

Comment:
Should we add some optional metrics?

Reply:
That's a good point. I am not sure about this change, I firstly enabled all optional metrics, but then I thought it would make sense to have the same configuration as in K8s. The upstream Helm chart does not enable any of the optional metrics, thus we would need to provide a custom hosmetrics receiver configuration in the opentelemetry-demo chart: https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-collector/templates/_config.tpl#L103

Given this context, would you still prefer to enable those optional metrics? (I don't have strong opinion)

Reply:
Hmmm, I think the main question here is if those metrics are important or not.
I'd say that at least the following 2 are:
- system.cpu.utilization
- system.memory.utilization

But I also don't have a strong opinion, this was more like a suggestion than a blocker on the PR.

Reply:
Sounds good, let's start with `system.cpu.utilization` and `system.memory.utilization` for now.